test/e2e/fundingManager/BondingCurveTokenRescueE2E.t.sol

```diff
@@ -51,13 +51,13 @@ contract BondingCurveTokenRescueE2E is E2ETest {
 
         // BancorFormula 'formula' is instantiated in the E2EModuleRegistry
 
-        IBondingCurveBase_v1.IssuanceToken memory issuanceToken_properties =
-        IBondingCurveBase_v1.IssuanceToken({
-            name: "Bonding Curve Token",
-            symbol: "BCT",
-            decimals: 18,
-            maxSupply: type(uint).max - 1
-        });
+        // IBondingCurveBase_v1.IssuanceToken memory issuanceToken_properties =
```

Comment:
This has been commented because of the compiler warning. As further up the file test file other parts were also out commented I went with this approach

Reply:
ah, it's never used, we define the issuance token twice and never use the first one. in that case I'd just remove it, no need to keep this around, the git history does this for us anyways.